packages/babel-helper-create-regexp-features-plugin/src/index.js

```diff
@@ -0,0 +1,96 @@
+import rewritePattern from "regexpu-core";
+import {
+  featuresKey,
+  FEATURES,
+  enableFeature,
+  runtimeKey,
+  hasFeature,
+} from "./features";
+import { generateRegexpuOptions } from "./util";
+
+import pkg from "../package.json";
+import { types as t } from "@babel/core";
+import { pullFlag } from "@babel/helper-regex";
+
+// Note: Versions are represented as an integer. e.g. 7.1.5 is represented
+//       as 70000100005. This method is easier than using a semver-parsing
+//       package, but it breaks if we release x.y.z where x, y or z are
+//       greater than 99_999.
+const version = pkg.version.split(".").reduce((v, x) => v * 1e5 + +x, 0);
```

Comment:
FWIW, I think it's perfectly fine using `semver` to parse (since we use it elsewhere)

Reply:
It's my fault 😛

Reply:
I think it is okay to leave it as is because all we need is a version sorter and we are not using `semver` elsewhere.